NLog.Targets.Http/HTTP.cs

```diff
@@ -225,6 +225,17 @@ public HTTP()
             HttpErrorRetryTimeout = 500;
         }
 
+
+        /// <inheritdoc />
+        protected override void CloseTarget()
+        {
+            var oldHttpClient = _httpClient;
```

Comment:
I don't understand the point here.

Reply:
Because `Dispose` should never throw, but I don't trust other peoples code, so I want the `_httpClient`-variable assigned to zero before calling `Dispose`.